apps/system/js/pin_page_system_dialog.js

```diff
@@ -128,7 +128,6 @@
     switch (action) {
       case 'pin':
         activeApp.appChrome.pinPage();
-        this._banner.show();
```

Comment:
Yeah I tried to remove that and things broke but I couldn't figure out why. It looks like a lot of the code used by the door hanger hasn't been cleaned up when the door hanger was removed.